File: PHP/Events/EventPerformances.php
<?php
namespace LTD_API;

class EventPerformances
{
    private $eventId;
    //array of Performance
    private $performances;
    
    public function getEventId()
    {
        return $this->eventId;
    }
    public function setEventId($eventId)
    {
        $this->eventId = $eventId;
    }

    public function getPerformances()
    {
        return $this->performances;
    }
    public function setPerformances($performances)
    {
        $this->performances = $performances;
    }
}

File: Java/Events/EventPerformances.java
package LTD_API;

import java.util.List;

public class EventPerformances
{
    private int eventId;
    private List<Performance> performances;
    
    public int getEventId()
    {
        return eventId;
    }
    public void setEventId(int eventId)
    {
        this.eventId = eventId;
    }

    public List<Performance> getPerformances()
    {
        return performances;
    }
    public void setPerformances(List<Performance> performances)
    {
        this.performances = performances;
    }
}

